content/en/continuous_integration/troubleshooting.md

```diff
@@ -101,6 +101,36 @@ This is done using the following algorithm:
 #### Possible issues with wall time calculation
 If you're using a library for testing time-dependent code, like [timecop][7] for Ruby or [FreezeGun][8] for Python, it is possible that test timestamps are wrong, and therefore calculated wall times. If this is the case, make sure that modifications to time are rolled back before finishing your tests.
 
+### The test status numbers are not what is expected
+
+The test status numbers are calculated based on the unique tests that were collected. The uniqueness of a test is defined not only by its suite and name, but by its test parameters and test configurations as well.
+
+#### The numbers are lower than expected
+
+If the numbers are lower than expected it is likely that either the library or the tool you are using to collect Test data cannot collect test parameters and/or some test configurations.
+
+1. If you are uploading JUnit test report files:
+  a. If you are running the same tests in different environment configurations, [make sure you are setting those configuration tags during the upload][9].
+  b. If you are running parameterized tests, it's very likely that the JUnit report does not have that information. [Try using a native library to report test data][10].
```

Comment:
Not sure why or how to fix it (maybe extra indentation?), but these a and b points are displayed in the same line in the preview

Reply:
Fixed!